src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: $white-color;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__img {
+    align-self: center;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-size: $main-font-size;
+    font-weight: $title-font-weight;
+  }
+
+  &__product-code {
+    margin-bottom: $basic-indent;
+
+    font-size: $secondary-font-size;
+    line-height: $secondary-line-height;
+    color: $secondary-color;
+  }
+
+  &__rewiews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-count {
+      font-size: $secondary-font-size;
+      line-height: $secondary-line-height;
+      color: $main-accent-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: $basic-indent;
+  }
+
+  &__price-label {
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    font-size: $accent-font-size;
+    font-weight: $accent-font-weight;
+  }
+
+  &__button {
+    height: 40px;
+    padding: 0;
+
+    font-size: $button-font-size;
+    font-weight: $accent-font-weight;
+    color: $white-color;
+    text-transform: uppercase;
+
+    background-color: $blue-accent-color;
+    border: none;
+    border-radius: 5px;
+    transition: 0.3s;
+    cursor: pointer;
+
+    &:hover {
+      color: $blue-accent-color;
+      background-color: $white-color;
+      border: 1px solid $blue-accent-color;
```

Comment:
do u can moved it to `button` class?
u use `box-sizing: border-box` on the whole project for a reason